### DISCUSSION TITLE
Realtime initial value

### DISCUSSION THREAD

**Discussion by clemorej** [pocketbase/pocketbase#1124] [Q&A] [answered]
How can I get the initial value of this record(Realtime)? Do I really have to create a separate request?

```
const pb = new PocketBase('http://127.0.0.1:8090');

pb.collection('projects').subscribe('RECORD_ID', function (e) {
			console.log(e.record);
		})
```

**Comment by ganigeorgiev** [comment, marked as answer]
Yes, you'll have to send a separate `getOne()` request.

The `subscribe` callback is triggered only on create/update/delete actions.
Otherwise it would be too ambiguous since it is unclear:
- what `e.action` would be in this case (currently the possible value is `create`, `update`, `delete`)
- what would be returned when you subscribe to the wildcard topic (aka. `*`)

**Reply by benallfree** [reply]
+1 for initial fetch. Current behavior is forces boilerplate code :)

> what e.action would be in this case (currently the possible value is create, update, delete)

I use a new status called `init`

> what would be returned when you subscribe to the wildcard topic (aka. *)

My wrapper accepts `initialFetch: boolean` 

IMHO the usefulness of the `*` topic is limited anyway until server-side filter expressions are accepted.
